Several minor changes to the OpenRC service:

* Execute the guix-daemon shipped by the Alpine package instead of
  the one installed to the profile of the root user.
* Don't log stdout and stderr of the guix-daemon by default, there
  is no really interesting information in these logs and they are
  not rotated as is.

To-Do:

* Optionally uset he guix-daemon from the root user's profile.
* Figure out if GUIX_LOCPATH and LC_ALL is really needed.

diff -upr guix-1.4.0.orig/etc/openrc/guix-daemon.in guix-1.4.0/etc/openrc/guix-daemon.in
--- guix-1.4.0.orig/etc/openrc/guix-daemon.in	2023-08-04 17:03:26.581703744 +0200
+++ guix-1.4.0/etc/openrc/guix-daemon.in	2023-08-04 17:03:49.168355599 +0200
@@ -19,10 +19,7 @@
 
 export GUIX_LOCPATH=@localstatedir@/guix/profiles/per-user/root/guix-profile/lib/locale
 export LC_ALL=en_US.utf8
-command="@localstatedir@/guix/profiles/per-user/root/current-guix/bin/guix-daemon"
+command="/usr/bin/guix-daemon"
 command_args="--build-users-group=guixbuild --discover=no"
 command_background="yes"
 pidfile="/var/run/guix-daemon.pid"
-
-output_log="/var/log/guix-daemon-stdout.log"
-error_log="/var/log/guix-daemon-stderr.log"
